### ISSUE TITLE
Support Battery Degradation Models from BLAST-Lite

### ISSUE THREAD

**Issue by WardLT** [ROVI-org/auto-soh#93]
**Is your feature request related to a problem? Please describe.**
We would like to be able to use or tweak the parameters of aging models from https://github.com/NREL/BLAST-Lite

**Describe the solution you'd like**
An ability to import an aging model from blast-lite with as minimal glue code as possible.

**Describe alternatives you've considered**
Keeping the math, but discarding the Python from BLAST-lite (which I don't want to do)

**Additional context**

**Comment by victorventuri** [comment]
Agreed, that would be nice, but we just need to remember that these should also be able to work in an online fashion.
